redbeams/src/main/java/com/sequenceiq/redbeams/converter/AbstractConversionServiceAwareConverter.java

```diff
@@ -0,0 +1,58 @@
+package com.sequenceiq.redbeams.converter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import org.springframework.core.convert.ConversionService;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.core.convert.converter.ConverterRegistry;
+
+import com.google.common.base.Strings;
+
+public abstract class AbstractConversionServiceAwareConverter<S, T> implements Converter<S, T> {
```

Comment:
This class is identical to the one in cloud-common. Unless the cloud-common copy is going away soon, this one is redundant and should be removed.

Reply:
Yes, you are right, I thought it was core ...